android/app/src/main/res/layout/fragment_search_no_tabs.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+  xmlns:tools="http://schemas.android.com/tools"
+  style="@style/MwmWidget.Floating"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+  <androidx.coordinatorlayout.widget.CoordinatorLayout
+    android:id="@+id/coordinator"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/app_bar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="?attr/colorPrimary"
+      android:elevation="@dimen/dp_0"
+      app:elevation="@dimen/dp_0">
+      <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar_with_search_no_elevation"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_scrollFlags="scroll|enterAlways"/>
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      app:layout_behavior="@string/appbar_scrolling_view_behavior">
+      <!-- Tabs -->
+      <LinearLayout
+        android:id="@+id/tab_frame_no_tabs"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <View
+          android:id="@+id/tabs_divider"
+          android:layout_width="match_parent"
+          android:layout_height="1dp"
+          android:background="?dividerHorizontal"/>
+
+        <androidx.viewpager.widget.ViewPager
+          android:id="@+id/pages_no_tabs"
+          style="@style/MwmWidget.Floating"
+          android:layout_width="match_parent"
+          android:layout_height="match_parent"/>
+      </LinearLayout>
+
+      <!-- Results -->
+      <FrameLayout
+        android:id="@+id/results_frame"
```

Comment:
Does this xml duplicate most of the items from the fragment with tabs? If yes, how can duplication be avoided? It may not be easy to keep two files in sync for every change.